src/pages/Welcome/index.js

```diff
@@ -1,12 +1,25 @@
 import React from 'react';
-import { CreateList } from '../../components/CreateList';
+import { CreateList } from '../../components/create-list';
+import { JoinList } from '../../components/join-list';
+import ContentContainer from '../../components/content-container';
+
+import './style.css';
 
 export const Welcome = () => (
-  <div className="container">
+  <ContentContainer>
     <h1>
```

Comment:
I'm not sure about [the Title component](https://github.com/the-collab-lab/tcl-34-smart-shopping-list/blob/main/src/components/title/index.js) purpose but I think it could be a good use case 😄  or why you didn't use it here.?

```suggestion
    <Title>
    Welcome to your <br />
      Smart Shopping list!
    </Title>
```